src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,68 @@
+package seedu.addressbook.ui;
+
+/**
+ * Formatter class to format text based on data structures
+ * @author Bernard Yip
+ *
+ */
+public class Formatter {
+    /** A decorative prefix added to the beginning of lines printed by AddressBook */
+    private static final String LINE_PREFIX = "|| ";
+
+    /** A platform independent line separator. */
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+
+    private static final String DIVIDER = "===================================================";
+
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+    /**
+     * Formats a string as a viewable indexed list item.
+     *
+     * @param visibleIndex visible index for this listing
+     */
+    public static String getIndexedListItem(int visibleIndex, String listItem) {
+        return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
+    }
+
+
+    /** Format message(s) to the user */
+    public static String formatToUser(String... message) {
+        StringBuffer displayString = new StringBuffer();
+        for (String m : message) {
+            displayString.append(LINE_PREFIX + m.replace("\n", LINE_SEPARATOR + LINE_PREFIX));
+            displayString.append(LINE_SEPARATOR);
+        }
+        return displayString.toString();
+    }
+
+    /**
+     * @return the linePrefix
```

Comment:
Based on the coding standards given, it would be more consistent for you to write a short description of what the method does. 
In this case `/**Returns the line prefix.*/` would be ideal. 
The `@return` param description can then be omitted as this method is trivial in execution.